lib/Models/Feature/Feature.ts

```diff
@@ -19,7 +19,7 @@ const customProperties = ["entityCollection", "properties", "data"];
 
 export default class TerriaFeature extends Entity {
   /** This object can be used to pass Terria-specific properties */
-  data?: TerriaFeatureData | JsonObject;
+  data?: TerriaFeatureData | JsonValue;
```

Comment:
I think the new type of more accurate - as I found examples where `data` was a string.

Either way, it will force us to do more explicit type checks before using `data` in the future

Reply:
OK sounds good